src/SmartComponents/SubscriptionsUtilized/SubscriptionsUtilizedCard.js

```diff
@@ -1,32 +1,138 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment/moment';
+import Immutable from 'seamless-immutable';
+import { Tooltip, TooltipPosition } from '@patternfly/react-core';
```

Comment:
need to be explicit imports, [kinda like this](https://github.com/RedHatInsights/insights-dashboard/blob/master/src/SmartComponents/Vulnerability/VulnerabilityCard.js#L2)